src/main/java/org/elasticsearch/action/update/UpdateRequest.java

```diff
@@ -205,105 +202,171 @@ int shardId() {
         return this.shardId;
     }
 
-    public String script() {
+    public Script scriptObject() {
```

Comment:
Not sure what to call this as there is already a method called script() which returns String and that needs to be maintained for bwc